plotly/tools.py

```diff
@@ -1269,3 +1276,547 @@ def return_figure_from_figure_or_data(figure_or_data, validate_figure):
             )
 
     return figure
+
+
+def Quiver(x, y, u, v,
```

Comment:
pep:8ball: you shouldn't Capitalize functions. https://www.python.org/dev/peps/pep-0008/#function-names